py/torch_tensorrt/dynamo/__init__.py

```diff
@@ -1,2 +1,5 @@
+from ._TorchTensorRTModule import TorchTensorRTModule
 from torch_tensorrt.dynamo import fx_ts_compat
-from .backend import compile
+from ._settings import *
+from torch_tensorrt.dynamo import backend
+from torch_tensorrt.dynamo.export import compile
```

Comment:
Maybe we could avoid directly promoting the `export.compile` function to the top level, as it could become quite confusing that `torch_tensorrt.dynamo.compile` invokes the `torch.export` path. The suggestion made here https://github.com/pytorch/TensorRT/discussions/1826#discussioncomment-6348562 could be helpful for an alternative.